From: Brian May <bam@debian.org>
Date: Sun, 6 Nov 2016 09:51:19 +1100
Subject: Update reference to python-werkzeug package

Point people towards installing the python-werkzeug package, not to
download random files from the internet.

Forwarded: not-needed
---
 django_extensions/management/commands/runserver_plus.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/django_extensions/management/commands/runserver_plus.py b/django_extensions/management/commands/runserver_plus.py
index beb1f00..4fdfc15 100644
--- a/django_extensions/management/commands/runserver_plus.py
+++ b/django_extensions/management/commands/runserver_plus.py
@@ -527,8 +527,7 @@ class Command(BaseCommand):
         if not HAS_WERKZEUG:
             raise CommandError(
                 "Werkzeug is required to use runserver_plus. "
-                "Please visit https://werkzeug.palletsprojects.com/ or install via pip."
-                " (pip install Werkzeug)"
+                "Please install `python3-werkzeug`"
             )
 
         # Set colored output
